ui/pages/confirmation/templates/index.js

```diff
@@ -33,6 +33,8 @@ const ALLOWED_TEMPLATE_KEYS = [
   'onSubmit',
   'networkDisplay',
   'submitText',
+  'loadingText',
+  'submitAlerts',
```

Comment:
Sorry for the confusion; I meant this to apply just to `'submitAlerts'`. The `'loadingText'` property is being used in the `add-ethereum-chain` template.

Reply:
@witmicko Looks like this is still unaddressed; this is the problem I referred to on Zoom.

It's not breaking the e2e test because the validation is broken (pre-existing issue, I'll address it now in a separate PR)